src/CarbonSystemSolvers.jl

```diff
@@ -207,6 +365,7 @@ end # module CarbonSolverApprox
 # ------------------------------------------------------------
 
 using .CarbonSystemSolvers.CarbonSolverApprox
```

Comment:
I didn't grasp earlier that "CarbonSolverApprox" was short hand for "approximate carbon solver". I suggest either `ApproximateCarbonSolver` for the name, or perhaps something more explicit like `CubicAlkalinityHydrogenSolver`